addons/popochiu/engine/objects/clickable/popochiu_clickable.gd

```diff
@@ -107,11 +106,10 @@ func _ready():
 	_translate()
 
 
-func _process(delta):
-	if Engine.is_editor_hint():		
-		if editing_polygon:
-			interaction_polygon = get_node("InteractionPolygon").polygon
-			interaction_polygon_position = get_node("InteractionPolygon").position
+func _notification(event):
```

Comment:
Add the type of the `event` parameter, and the return type of the function:

```suggestion
func _notification(event: int) -> void:
```

By the way, I like this approach.